markets/perps-market/test/integration/helpers/collateralHelper.ts

```diff
@@ -1,23 +1,17 @@
 import { ethers } from 'ethers';
 import { Systems } from '../bootstrap';
+import { SynthMarkets } from '@synthetixio/spot-market/test/common';
 
-type IncomingChainState = {
-  systems: () => Systems;
-  provider: () => ethers.providers.JsonRpcProvider;
-};
-
-type TradeSynthData = {
-  synthName: () => string;
-  synthMarketId: () => ethers.BigNumberish;
-  synthMarket: () => ethers.Contract;
-  marginAmount: () => ethers.BigNumberish;
-  synthAmount: () => ethers.BigNumberish;
+type CollateralSynthData = {
+  synthMarket?: () => SynthMarkets[number];
+  snxUSDAmount: () => ethers.BigNumber;
 };
 
-export type SetCollateralData = {
+export type DepositCollateralData = {
+  systems: () => Systems;
   trader: () => ethers.Signer;
   accountId: () => number;
-  trades: TradeSynthData[];
+  collaterals: CollateralSynthData[];
```

Comment:
renamed. Changed from trades to collaterals since we are not always trading (i.e. snxUSD as collateral)